app/build.gradle

```diff
@@ -28,11 +28,11 @@ dependencies {
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
     compile 'com.android.support:design:25.3.1'
```

Comment:
so ive had an error for the past 30 mins that i havent been able to work out.... it was deleting this that caused it :P